### DISCUSSION TITLE
Will u provide explanation how to upgrade to new version of your boilerplate?

### DISCUSSION THREAD

**Discussion by michaelpautov** [obytes/react-native-template-obytes#282] [Docs Comments]
Will u provide explanation of how to upgrade to new version of your boilerplate?

**Comment by yjose** [comment]
Hey @michaelpautov, unfortunately, we don't have an upgrade guide. The only way is to check the update based on Git releases, like the following:  https://github.com/obytes/react-native-template-obytes/compare/v5.1.0...v5.2.0

you can check your project `package.json` file  and check  `osMetadata.initVersion`  and then compare it with the last starter release
